app/views/admin/index.html.haml

```diff
@@ -3,6 +3,9 @@
 .post-title-divider
   %span
 .entry-content
+
+  = button_to t('.export'), admin_export_ansokan_csv_url, {class: 'btn export-ansokan'}
+
   %table
     %thead
       %tr
```

Comment:
Most of this file is duplicated in views/membership_applications.index.html.haml.  Perhaps we could create common partial(s) for the same content?

Or, use views/membership_applications.index.html.haml only and make the "export" button visibility conditional on the user being admin?

Or, maybe these views will differ more as time goes on and thus it makes sense to keep separate?  (at the very least, I would suggest a shared partial for the applications table.